File: CodingAcademy_Programs/MongoDB/MongoDB_2/select.rb
require 'rubygems'
require 'mongo'

include Mongo

$client = Mongo::Client.new([ '127.0.0.1:27017' ], :database => 'LXF')
Mongo::Logger.logger.level = ::Logger::ERROR
$collection = $client[:someData]

$collection.find({"n" => {"$gt" => 205000} ,"code" => {"$lt" => 1}}).each do |doc|
  puts doc
end

File: CodingAcademy_Programs/MongoDB/MongoDB_2/select.py
from pymongo import MongoClient
client = MongoClient("127.0.0.1", 27017)
db = client.LXF

# Choose a collection
someData = db.someData

for document in someData.find():
    print document

for doc in someData.find({"code": {"$gt": 30}}).sort("n"):
    print doc